Gruntfile.js

```diff
@@ -87,7 +87,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-jsxgettext');
   grunt.loadNpmTasks('grunt-zanata-js');
 
-  grunt.registerTask('test', ['fh:test']);
+  grunt.registerTask('test', ['eslint','fh:test']);
```

Comment:
I think that test is not standard task we support and use in CI. Basic tasks (backed by grunt plugin are unit and accept). This change should remove jshint